curvefs/src/client/inode_cache_manager.cpp

```diff
@@ -237,21 +238,36 @@ CURVEFS_ERROR InodeCacheManagerImpl::BatchGetInodeAttr(
 
 CURVEFS_ERROR InodeCacheManagerImpl::BatchGetInodeAttrAsync(
     uint64_t parentId,
-    const std::set<uint64_t> &inodeIds,
+    std::set<uint64_t> *inodeIds,
     std::map<uint64_t, InodeAttr> *attrs) {
-    if (inodeIds.empty()) {
-        return CURVEFS_ERROR::OK;
-    }
-
     NameLockGuard lg(asyncNameLock_, std::to_string(parentId));
     bool ok  = iAttrCache_->Get(parentId, attrs);
     if (ok) {
         return CURVEFS_ERROR::OK;
     }
 
+    // get some inode attr in icache
```

Comment:
Should we get attrs from `iCache_` first, then `iAttrCache_`? In other words, can `iAttrCache_` has outdated attributes in the same scenario?

Reply:
fixed